LibertyDSNP/frequency
# fix: MSA owner should be able to invoke non-signature methods on SignatureRequired schemas directly

# Goal
The goal of this PR is to make it possible for MSA owners to call non-signature stateful-storage methods directly without requiring a separate payload signature.

Adds e2e test scenarios to validate this correct behavior for both signature and non-signature stateful-storage extrinsics

Closes #1793 

# Discussion
## Testing
* Run e2e tests for the `stateful-storage` pallet against an unmodified frequency chain; should result in ~ 6 errors
* Run e2e tests against a locally-built chain from this branch; errors should be resolved

# ~TODO~
- [x] Add Rust unit tests

# Checklist
- [ ] Weights updated
